src/cli.ts

```diff
@@ -194,13 +194,13 @@ function transformMarkdown(buf: Buffer) {
         if (/^( {4}|\t)/.test(line) && prevLineIsEmpty) {
           output.push(line)
           state = 'tab'
-        } else if (/^```(js|javascript)$/.test(line)) {
+        } else if (/^(```|~~~)(js|javascript)$/.test(line)) {
```

Comment:
This does not solve the issue: a code block is started by three *or more* of either `` ` `` or `~` characters and ended by *the same sequence*. First the pattern must be `` (```+|~~~+) `` and second the regex capturing group must be stored to detect the end of the code block.

Reply:
Good point.

Reply:
@nichtich @antonmedv Thank you for the review! I applied it to this PR.